leetcode/src/69.c

```diff
@@ -0,0 +1,24 @@
+//using binary search method is one of the efficient one for this problem statement.
+int mySqrt(int x){
+int start=0;
+        int end=x;
+        long long int ans=0;
+        while(start <= end){
+               long long int mid=(start+end)/2;
+               if(mid*mid == x){
+                         ans=mid;
```

Comment:
May just `return mid;`

Reply:
sorry I can't return mid because when mid*mid < number  we need to store the max value of mid so I used ans to store the value of the mid.But i will return mid for the case when mid*mid == number .
thank you for the suggestions!!!